test/e2e/flowaggregator_test.go

```diff
@@ -532,6 +504,102 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 			checkRecordsForFlows(t, data, podAIPs.ipv4.String(), svcC.Spec.ClusterIP, isServiceIPv6, false, true, false, false, checkBandwidth)
 		}
 	})
+
+	// testFlowAggregatorAntctl ensures antctl is accessible in a Flow Aggregator Pod
+	// and checks the output of antctl commands.
+	t.Run("testFlowAggregatorAntctl", func(t *testing.T) {
+		skipIfNotRequired(t, "mode-irrelevant")
+		flowAggPod, err := data.getFlowAggregator()
+		if err != nil {
+			t.Fatalf("Error when getting flow-aggregator Pod name: %v", err)
+		}
+		podName := flowAggPod.Name
+		for _, args := range antctl.CommandList.GetDebugCommands(runtime.ModeFlowAggregator) {
+			command := []string{}
+			if testOptions.enableCoverage {
+				antctlCovArgs := antctlCoverageArgs("antctl-coverage")
+				command = append(antctlCovArgs, args...)
+			} else {
+				command = append([]string{"antctl", "-v"}, args...)
+			}
+			t.Logf("command: %s", command)
```

Comment:
It's better to format the log like: "run command: %s" and pass command as string.

Reply:
Updated. Still use the array format command to align with the agent and controller side as discussed below, Thanks!